src/vsr/client.zig

```diff
@@ -734,9 +764,7 @@ pub fn ClientType(
             assert(!self.request_timeout.ticking);
             self.request_timeout.start();
 
-            // If our view number is out of date, then the old primary will forward our request.
-            // If the primary is offline, then our request timeout will fire and we will
-            // round-robin.
+            // Otherwise we know the view and send directly to the primary.
```

Comment:
This "Otherwise" feels like it is missing some context.

Reply:
Ah, that's the remnant of "broadcast .register" code which I decided to take out for now!